.changeset/old-students-push.md

```diff
@@ -0,0 +1,12 @@
+---
+"@justeattakeaway/pie-tag": minor
+"pie-storybook": minor
+"pie-docs": minor
+---
+
+[Updated] - (Breaking change) rename the following variants:
```

Comment:
does this need to be a major bump @raoufswe ? If so we also need to major bump the webc package

Reply:
if we do so, then the component will go to 1.0.0 right? are we fine with that. the component is currently on beta

Reply:
updated to major